restli-tools/src/main/java/com/linkedin/restli/tools/symbol/SymbolTableNameHandler.java

```diff
@@ -56,26 +56,37 @@
    */
   String generateName(List<String> symbols)
   {
+    if (_serverNodeUri == null)
+    {
+      throw new IllegalStateException("Cannot generate symbol table name with null server node URI.");
+    }
+
     return _serverNodeUri + SERVER_NODE_URI_PREFIX_TABLENAME_SEPARATOR + _symbolTablePrefix
         + PREFIX_HASH_SEPARATOR + symbols.hashCode();
   }
 
   @Override
   protected SymbolTableMetadata createMetadata(String serverNodeUri, String tableName) {
     // A table is remote if the server node URI does not match the current server node URI.
-    boolean isRemote = !_serverNodeUri.equals(serverNodeUri);
+    boolean isRemote = _serverNodeUri == null || !_serverNodeUri.equals(serverNodeUri);
```

Comment:
So how come if serverNodeUri is null then this thing is remote?
If serverNodeUri is null, could it mean something else, like not properly configuration?